File: WinPaletter/Dialogs/LicenseForm.vb
Public Class LicenseForm
    Private Sub LicenseForm_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        LoadLanguage
        ApplyStyle(Me)
        TextBox1.Font = My.Application.ConsoleFontLarge
        TextBox1.Text = My.Resources.LICENSE
        My.Computer.Audio.PlaySystemSound(Media.SystemSounds.Exclamation)
    End Sub

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        DialogResult = DialogResult.OK
        My.Settings.General.LicenseAccepted = True
        My.Settings.General.Save()
        Me.Close()
    End Sub

    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click
        DialogResult = DialogResult.Cancel
        My.Settings.General.LicenseAccepted = False
        My.Settings.General.Save()
        Using Prc As Process = Process.GetCurrentProcess : Prc.Kill() : End Using
    End Sub
End Class

File: WinPaletter/Dialogs/LicenseForm.cs
using System;
using System.Diagnostics;
using System.Windows.Forms;

namespace WinPaletter
{
    public partial class LicenseForm
    {
        public LicenseForm()
        {
            InitializeComponent();
        }
        private void LicenseForm_Load(object sender, EventArgs e)
        {
            this.LoadLanguage();
            ApplyStyle(this);
            TextBox1.Font = Fonts.ConsoleLarge;
            TextBox1.Text = Properties.Resources.LICENSE;
            Program.Computer.Audio.PlaySystemSound(System.Media.SystemSounds.Exclamation);
        }

        private void Button1_Click(object sender, EventArgs e)
        {
            DialogResult = DialogResult.OK;
            Program.Settings.General.LicenseAccepted = true;
            Program.Settings.General.Save();
            Close();
        }

        private void Button2_Click(object sender, EventArgs e)
        {
            DialogResult = DialogResult.Cancel;
            Program.Settings.General.LicenseAccepted = false;
            Program.Settings.General.Save();
            using (var Prc = Process.GetCurrentProcess())
            {
                Prc.Kill();
            }
        }
    }
}